### DISCUSSION TITLE
Public bevy sponsors avatars

### DISCUSSION THREAD

**Discussion by ThierryBerger** [bevyengine/bevy#5309] [Ideas]
# Context
cart had a sponsor goal which would include bevy avatars as a perk. 
Those are tailor made for each sponsor and are "skins" for bevy logo.

<details>
<summary>
Initial Request description
</summary>

![image](https://user-images.githubusercontent.com/2290685/178747669-167077f0-db05-410a-a3b5-2e40874fcfc1.png)

@mockersf expressed on discord he was ok with sharing it https://discord.com/channels/691052431525675048/745355529777315850/950492865086255184 ; and that image was shared in discord along with a few others.

# Idea

I thought those images would make neat ecosystem images for game jams, examples, So I'm asking for them to be public.

- I'm not sure about the license they should have ? I'd like to at least use them in non commercial games 🤷

cart expressed on discord he was willing to make them public if sponsors were OK: https://discord.com/channels/691052431525675048/745355529777315850/950492707145539634.

</details>

# 🎉 Victory 🎉 

Check out https://thebevyflock.github.io/the-bird/ (thanks @mockersf !)

Cart and all 5 owners of the bevy avatars agreed to share theirs in CC-0 🎉 !

Here is the source of the 5 bevy avatars, shared by cart [from discord](https://discord.com/channels/691052431525675048/745355529777315850/950173023971999804): 

![Bevy Bird Sponsors](https://github.com/bevyengine/bevy/assets/2290685/10039481-d07c-4e7d-8f8b-9adba0bcc540)

<details><summary>Agreements list</summary>
<p>

MGlolenstine, Sodacodery, mockersf: https://discord.com/channels/691052431525675048/745355529777315850/950492707145539634
alexhartstone: https://discord.com/channels/691052431525675048/745355529777315850/950492954533965914
joshuajbouw: https://github.com/bevyengine/bevy/discussions/5309#discussioncomment-3700490

</p>
</details>

**Comment by mockersf** [comment]
As already said, OK for me!

Cart at the time proposed a CC-0 license, maybe a CC-BY-4.0 (and asking to mention "Bevy") would be nice?

**Reply by cart** [reply]
Hmm I think I prefer CC-0, just to make it easier to use them worry free. Given that they're "bevy birds" I doubt they'll have much appeal outside of the Bevy ecosystem, so requiring an attribution clause will likely be mostly redundant. 

But each "bird owner" is free to choose whatever license they want, as it is "their" bird.

**Comment by joshuajbouw** [comment]
I'm fine with a CC-0 license for mine.

**Comment by ThierryBerger** [comment]
Thanks @mockersf for helping me realize we had the whole list public and all relevant parties had agreed to the CC-0 licencing !

I've updated the discussion description.
